modules/hello-world/pages/start-using-ottoman.adoc

```diff
@@ -0,0 +1,217 @@
+= Install and Start Using the Ottoman ODM with Couchbase Server
+:description: Installing the Ottoman ODM & a Hello World program.
+:navtitle: Start Using the Ottoman ODM
+
+include::project-docs:partial$attributes.adoc[]
+
+[abstract]
+Ottoman is an open-source Object Document Mapper(ODM) library, built for Node.js and Couchbase, that aspires to give developers an even better experience when building modern applications.
```

Comment:
Should this text be the first paragraph, with the :description: text being the abstract?

Reply:
Hmm, it *does* also make sense as the initial paragraph. My thinking was that the description text seemed quite short as an abstract... 🤔 I'll make the changes 👍 .